internal/db/schema/migrations/oss/postgres/91/05_deletion_tables_view.up.sql

```diff
@@ -0,0 +1,15 @@
+-- Copyright (c) HashiCorp, Inc.
+-- SPDX-License-Identifier: BUSL-1.1
+
+begin;
+  -- Originially added in 81/01_deleted_tables_and_triggers.up.sql
```

Comment:
We usually note in the old migration where this was replaced

Reply:
Good catch, added a comment in the old migration.